tools.go

```diff
@@ -19,6 +19,6 @@ import (
 	_ "github.com/kisielk/errcheck"
 	_ "github.com/securego/gosec"
 	_ "golang.org/x/lint/golint"
-	_ "golang.org/x/tools"
```

Comment:
In go1.13 this will cause an error when running `go mod tidy`. It may be because the top-level packages do not contain any Go files